CHANGELOG.adoc

```diff
@@ -7,6 +7,11 @@
 
 == DFX
 
+=== fix: Freezing Threshold now documented
+
+Calls made to retrieve the help output for `canister update-settings` was missing the `freezing-threshold` parameter.
+This is now included when running `dfx canister update-settings -h`.
+
```

Comment:
This actually needs to go in the "UNRELEASED" section (with a new `== DFX` header section)